### DISCUSSION TITLE
Expose an option to update light sliders on release or on drag.

### DISCUSSION THREAD

**Discussion by fcassata** [Clooos/Bubble-Card#817] [Feature requests]
I know some people were asking for this change implemented in the latest beta, but it makes no sense to me. When we set our lights we don't know exactly where in the range of each slider we want to set it, but we know what light level we want. "Live" updates make this intuitive and easy, update on release makes it a guessing game. Just think about physical dimmers, they don't "update" when you stop turning, they update "live" so you know when to stop.

**Comment by Clooos** [comment]
Hi, I will add the possibility to revert that behavior 👌

**Reply by fcassata** [reply]
YAY, thanks for considering it :)

**Reply by Clooos** [reply]
<img width="569" alt="image" src="https://github.com/user-attachments/assets/3748a13a-7eee-4382-a10d-cbd7f225d2b7">

Just wait for the next beta now 😁

**Reply by Clooos** [reply]
The new beta is here:

https://github.com/Clooos/Bubble-Card/releases/tag/v2.3.0-beta.2

**Reply by fcassata** [reply]
You are fast! Will test!
